examples/benchmark.py

```diff
@@ -184,95 +190,121 @@ def benchmark_train_and_valid(base_dir, train_dataset, valid_dataset, tasks,
-    print('Start fitting by tensorflow')
-    model_robust.fit(train_dataset,nb_epoch = nb_epoch)
-    
+    print('--------------------------------------------')
+    print('Start fitting by robust multitask DNN')
+    model_robust.fit(train_dataset, nb_epoch=nb_epoch)
+
+    # Evaluating tensorflow robust MultiTaskDNN model
     train_scores['tf_robust'] = model_robust.evaluate(
         train_dataset, [classification_metric], transformers)
 
-    valid_scores['tf_robust'] = model_robust.evaluate( 
+    valid_scores['tf_robust'] = model_robust.evaluate(
         valid_dataset, [classification_metric], transformers)
 
-
   if model == 'logreg':
-    # Building tensorflow logistic regression model
-    learning_rate = hyper_parameters['learning_rate']
+    # Loading hyper parameters
     penalty = hyper_parameters['penalty']
     penalty_type = hyper_parameters['penalty_type']
     batch_size = hyper_parameters['batch_size']
     nb_epoch = hyper_parameters['nb_epoch']
+    learning_rate = hyper_parameters['learning_rate']
 
-    model_logreg = dc.models.TensorflowLogisticRegression(
-          len(tasks), n_features, learning_rate=learning_rate, penalty=penalty, 
-          penalty_type=penalty_type, batch_size=batch_size, 
-          seed=seed, verbosity=verbosity)
- 
+    # Building tensorflow logistic regression model
+    model_logreg = dc.models.TensorflowLogisticRegression(len(tasks),
+        n_features, penalty=penalty, penalty_type=penalty_type, 
+        batch_size=batch_size, learning_rate=learning_rate, 
+        verbosity=verbosity)
+    
     print('-------------------------------------')
     print('Start fitting by logistic regression')
-    model_logreg.fit(train_dataset,nb_epoch = nb_epoch)
+    model_logreg.fit(train_dataset, nb_epoch=nb_epoch)
     
-    train_scores['logreg'] = model_logreg.evaluate(train_dataset,
-                               [classification_metric],transformers)
-
-    valid_scores['logreg'] = model_logreg.evaluate(valid_dataset,
-                               [classification_metric],transformers)
+    # Evaluating tensorflow logistic regression model
+    train_scores['logreg'] = model_logreg.evaluate(
+        train_dataset, [classification_metric], transformers)
 
+    valid_scores['logreg'] = model_logreg.evaluate(
+        valid_dataset, [classification_metric], transformers)
+    
   if model == 'graphconv':
     # Initialize model folder
     model_dir_graphconv = os.path.join(base_dir, "model_graphconv")
-    
-    
+
+    # Loading hyper parameters
+    batch_size = hyper_parameters['batch_size']
+    nb_epoch = hyper_parameters['nb_epoch']    
     learning_rate = hyper_parameters['learning_rate']
     n_filters = hyper_parameters['n_filters']
     n_fully_connected_nodes = hyper_parameters['n_fully_connected_nodes']
-    batch_size = hyper_parameters['batch_size']
-    nb_epoch = hyper_parameters['nb_epoch']
-    
+    seed = hyper_parameters['seed']
```

Comment:
I think `seed` shouldn't be a hyperparameter. It should just be a number like (123) that ensures that all users run tests under the same conditions.